node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminStateUpdaterImpl.java

```diff
@@ -209,10 +209,11 @@ void tick() {
                 // To avoid node agents stalling for too long, we'll force unfrozen ticks now.
                 log.info("Timed out trying to freeze, will force unfreezed ticks");
                 nodeAdmin.setFrozen(false);
+                fetchContainersToRunFromNodeRepository();
```

Comment:
Logically it seems better to put this before setFrozen?

Reply:
Reversed order.